configs/experiment_schema.json

```diff
@@ -0,0 +1,118 @@
+{
+  "title": "Experiment schema",
+  "type": "object",
+  "minProperties": 1,
+  "additionalProperties": false,
+  "properties": {
+    "rounds":{
```

Comment:
I think we should add some predefined optional properties:

* description
* contact

And possibly other general-purpose fields.